### DISCUSSION TITLE
'USE_NGINX_CONF_FILES=true' mentioned but there is nothing like that in '/.env'

### DISCUSSION THREAD

**Discussion by datenheim** [evertramos/nginx-proxy-automation#304] [Q&A] [answered]
While stumbling over nginx' default `client_max_body_size` being only 1MiB I found out that one should do (in /conf.d/uploadsize.conf for example):
`# [WARNING] To enable this files you need to uncomment USE_NGINX_CONF_FILES=true in .env file`

But then I found neither in the `/.env.sample` nor `/.env` after fresh install such a comment does exist.

1. If I add `USE_NGINX_CONF_FILES=true` to the `/.env` file after the proxy is already running, what to do next to fix the upload size?
2. Shouldn't the mentioned comment exist in the `/.env.sample` and `/.env`?

After writing this I found #261 and #268 but at least 2. might be added to the provided `.env.sample` or `.env` to be consistent with the comments in `/conf.d/*.conf`

**Comment by evertramos** [comment, marked as answer]
@datenheim 

Sorry I missed that update in the comments... Just update that file and here is the correct answer for that:

#309

**Comment by tomritsema** [comment]
For people entering this answer from a internet search:
The configuration option to specify that nginx config files should be used (USE_NGINX_CONF_FILES=true in .env) has been replaced by application option --use-nginx-conf-files to the script 'fresh-start.sh'

**Reply by evertramos** [reply]
That is correct! 

p.s.: I tried to mark this as an anwser but couldn´t...
